game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/change/RemoveUnprotectedUnits.java

```diff
@@ -52,6 +53,7 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
     removeUnprotectedUnits(bridge, BattleState.Side.OFFENSE);
   }
 
+  @RemoveOnNextMajorRelease("This doesn't need to be public in the next major release")
```

Comment:
:+1: 

This looks good. We've had issues of marking changes that needed to happen on the next major release and have missed some or come across them and were then unsure of what to change exactly. It looks like this annotation is working out, I'm much more confident on next major we'll catch & fix these types of things.